cmd/kusk/cmd/dashboard.go

```diff
@@ -54,23 +55,41 @@ The flags --envoyfleet.namespace and --envoyfleet.name can be used to change the
 	Expose dashboard on port 9090
 	`,
 	Run: func(cmd *cobra.Command, args []string) {
+		reportError := func(err error) {
+			if err != nil {
+				// Report error
+				miscInfo := map[string]interface{}{
+					"envoyfleet.namespace": dashboardEnvoyFleetNamespace,
+					"envoyfleet.name":      dashboardEnvoyFleetName,
+					"external-port":        dashboardEnvoyFleetExternalPort,
+					"args":                 args,
+					"os.Args":              os.Args,
+					"config":               cfgFile,
```

Comment:
Also this is feeding too much data that's not really useful for us

Reply:
Please mark resolved if you're happy with it.